packages/terra-form-select/src/single/Frame.jsx

```diff
@@ -248,6 +248,17 @@ class Frame extends React.Component {
       return;
     }
 
+    // eslint-disable-next-line no-underscore-dangle
+    const _onBlur = () => {
+      this.setState({ isFocused: false });
+
+      this.closeDropdown();
+
+      if (this.props.onBlur) {
+        this.props.onBlur(event);
+      }
+    };
```

Comment:
Is this only an issue with the single select variant?

Reply:
Yes.

Reply:
Can you explain why only this variants needs the special onBlur logic?

Reply:
Sure. The existing inline code comment explains it - I noticed that unlike the default variant, the blur event does not get triggered when one's trying to open the dropdown menu on the other variants. This special logic takes care of suppressing the rest of the blur logic by not closing the dropdown menu immediately upon Frame click in the default variant.

https://github.com/cerner/terra-core/blob/22e6cb5e503a02e8242061730274ff3bd1c61eb4/packages/terra-form-select/src/single/Frame.jsx#L251-L255